theme/layouts/_head.ejs

```diff
@@ -18,7 +18,11 @@
 
 <!-- Resources -->
 <script src="<%= site.root %>/js/main.js" defer></script>
-<link href="<%= site.root %>/css/main.css" rel="stylesheet">
+<% if (page.pathname == "") { %>
```

Comment:
Suggestion: As we are applying Layout related style and it's just in our specific case that lyout happens to be index page, what about using:
```
page.layout == "homepage"
```

Instead?